environment.yml

```diff
@@ -42,4 +43,6 @@ dependencies:
   - sphinx-click
   - pip:
     - sphinx-toolbox
+    - espaloma_charge
```

Comment:
espaloma is quite a heavy.. even if it's for testing, I could live with either not testing it or removing the code path completely if we're not happy with it (although it does look promising as a method)

Reply:
Note: I am not advocating for either nagl or espaloma to be ACTUAL depdencies of OpenFE. I.e. on conda-forge these should not be there! This should be purely optional code that folks can use if they install the relevant things.